src/corefx/System.Globalization.Native/collation.cpp

```diff
@@ -331,13 +331,19 @@ extern "C" int32_t GlobalizationNative_GetSortVersion()
     return UCOL_RUNTIME_VERSION << 16 | UCOL_BUILDER_VERSION;
 }
 
-extern "C" SortHandle* GlobalizationNative_GetSortHandle(const char* lpLocaleName)
+extern "C" SortHandle* GlobalizationNative_GetSortHandle(const char* lpLocaleName, int32_t *errorCode)
```

Comment:
I believe that the typical pattern in corefx is to return the actual result via out argument; and the error code as return.